cmd/podman/save.go

```diff
@@ -70,6 +71,10 @@ func saveCmd(c *cliconfig.SaveValues) error {
 		return errors.Errorf("need at least 1 argument")
 	}
 
+	if len(args) > 1 {
+		logrus.Errorf("only the first image will be saved but %d are specified", len(args))
+	}
```

Comment:
`LocalRuntime.SaveImage` uses `c.inputArgs[1:]` as `DockerArchiveAdditionalTags`. That might be not what the command is _expected_ to do — but this breaks that.

So, either that functionality is useful, and this error needs to be removed (or reworded, or… I don’t know), or it is not, and it should be removed from the code. I guess the former.